cafy_pytest/plugin.py

```diff
@@ -515,49 +515,63 @@ def pytest_configure(config):
                 del os.environ["hybrid_mode"]
 
         #Debug Registration Server code
-        if CLS and int(CLS):
-            registration_id = os.environ.get("REG_ID", None)
-            CafyLog.registration_id = registration_id 
+        #Cant introduce switch case as this plugin is common for 3.6 and 3.11 env.
+        call_registeration = False
+        if CLS and not cafykit_debug_enable :
```

Comment:
If we change the behavior here, we must change the CLS definition in line 58 to `cd /auto/tftp-niupadhy-sjc/cls/311_env/bin/`

Reply:
Did the change at line 58 and added comments how to improve it next iteration.